### PR TITLE
Swagger and examples for Microsoft.DataBoxEdge RP

### PR DESCRIPTION
**Pull request by anponnet**
Swagger and examples for Microsoft.DataBoxEdge RP

### CODE REVIEW

File: specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2018-07-01/edgedateway.json [hunk 1]
```diff
@@ -0,0 +1,3256 @@
...
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/AlertProperties",
+          "description": "Properties of alert.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "AlertErrorDetails": {
+      "description": "Errordetails for the alert.",
```
Comment (nschonni) [inline specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2018-07-01/edgedateway.json]: Errordetails -> Error details

File: specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2018-07-01/edgedateway.json [hunk 2]
```diff
@@ -0,0 +1,3256 @@
...
+          "enum": [ "Informational", "Warning", "Critical" ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "AlertSeverity",
+            "modelAsString": false
+          }
+        },
+        "errorDetails": {
+          "$ref": "#/definitions/AlertErrorDetails",
+          "description": "ErrorDetails of the alert."
```
Comment (nschonni) [inline specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2018-07-01/edgedateway.json]: Errordetails -> Error details

File: specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2018-07-01/edgedateway.json [hunk 3]
```diff
@@ -0,0 +1,3256 @@
...
+      "properties": {
+        "errorCode": {
+          "description": "Error code.",
+          "type": "string"
+        },
+        "errorMessage": {
+          "description": "Error Message.",
+          "type": "string"
+        },
+        "occurences": {
```
Comment (nschonni) [inline specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2018-07-01/edgedateway.json]: occurences -> occurrences

File: specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2018-07-01/edgedateway.json [hunk 4]
```diff
@@ -0,0 +1,3256 @@
...
+          "format": "int32",
+          "description": "Number of occurences.",
```
Comment (nschonni) [inline specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2018-07-01/edgedateway.json]: occurences -> occurrences

File: specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2018-07-01/edgedateway.json [hunk 5]
```diff
@@ -0,0 +1,3256 @@
...
+        }
+      }
+    },
+    "BandwidthScheduleProperties": {
+      "description": "The properties of the bandwidth schedule.",
+      "required": [ "start", "stop", "rateInMbps", "days" ],
+      "type": "object",
+      "properties": {
+        "start": {
+          "description": "The start time of the schdule in UTC.",
```
Comment (nschonni) [inline specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2018-07-01/edgedateway.json]: schdule -> schedule

File: specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2018-07-01/edgedateway.json [hunk 6]
```diff
@@ -0,0 +1,3256 @@
...
+          "type": "string"
+        },
+
```
Comment (nschonni) [inline specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2018-07-01/edgedateway.json]: aggreagtion -> aggregation

Comment (nschonni) [inline specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2018-07-01/edgedateway.json]: Accessprotocol -> Access protocol

Comment (nschonni) [inline specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2018-07-01/edgedateway.json]: atleast -> at least

Comment (nschonni) [inline specification/edgegateway/resource-manager/Microsoft.DataBoxEdge/stable/2018-07-01/edgedateway.json]: installl -> install

**Review by sarangan12 — CHANGES_REQUESTED** [review]

**Review by anponnet — COMMENTED** [review]
Changes requested are made and pushed

**Review by sarangan12 — APPROVED** [review]

**Pull request merged**